### PR TITLE
Remove bundle dependencies to use with Silex

### PR DESCRIPTION
**Pull request by GromNaN**
I resubmit PR #238 because it was not updated by the new push.

The Security and SessionPersistance classes can be used outside of the fullstack framework.

Example: https://github.com/GromNaN/FacebookServiceProvider

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,12 +18,14 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": ">=2.1,<2.3",
-        "symfony/security-bundle": ">=2.1,<2.3",
-        "symfony/twig-bundle": ">=2.1,<2.3",
+        "symfony/http-foundation": ">=2.1,<2.3",
```
Comment (GromNaN) [inline composer.json:21]: This is informative as `symfony/security` requires `symfony/http-foundation`

**Comment by diegoholiveira** [comment]
Can you provide the proper documentation about how to integrate it with silex?

**Comment by GromNaN** [comment]
@diegoholiveira I've created a service provider : https://github.com/GromNaN/FacebookServiceProvider#readme

**Comment by diegoholiveira** [comment]
Nice.

Do you think on merge this service provider under FOSFacebookBundle?

**Comment by GromNaN** [comment]
I'm not against moving my FacebookServiceProvider to the FOS namespace.

But for the inclusion into the bundle, I see pro & con:

Pro:
- The provider and the bundle development are synchronized.
- Unit tests prevent a change on the bundle to break the provider
- Twig helper can be implemented more easily (path to the view is fixed inside the bundle)

Con:
- Lose visibility as the provider is only 1 file in the bundle. People looking for facebook connect solution for Silex will miss it.
- The provider may require uptades to follow Silex BC breaks, and require new tags even if the bundle itself has not been modified.

**Pull request merged**